db/setup/db-mock-seeds-population/db-mock-seeds-population.py

```diff
@@ -0,0 +1,73 @@
+import uuid
+import psycopg
+import os
+
+NACHET_DB_URL = os.getenv("NACHET_DB_URL")
+
+NACHET_SCHEMA = os.getenv("NACHET_SCHEMA")
+
+# Connect to your PostgreSQL database with the DB URL
+conn = psycopg.connect(NACHET_DB_URL)
+# Create a cursor object
+cur = conn.cursor()
+
+brassicaNapus = (str(uuid.uuid4()), "Brassica napus")
+brassicaJunsea = (str(uuid.uuid4()), "Brassica juncea")
+cirsiumArvense = (str(uuid.uuid4()), "Cirsium arvense")
+cirsiumVulgare = (str(uuid.uuid4()), "Cirsium vulgare")
+carduusNutans = (str(uuid.uuid4()), "Carduus nutans")
+bromusSecalinus = (str(uuid.uuid4()), "Bromus secalinus")
+bromusHordeaceus = (str(uuid.uuid4()), "Bromus hordeaceus")
+bromusJaponicus = (str(uuid.uuid4()), "Bromus japonicus")
+loliumTemulentum = (str(uuid.uuid4()), "Lolium temulentum")
+solanumCarolinense = (str(uuid.uuid4()), "Solanum carolinense")
+solanumNigrum = (str(uuid.uuid4()), "Solanum nigrum")
+solanumRostratum = (str(uuid.uuid4()), "Solanum rostratum")
+ambrosiaArtemisiifolia = (str(uuid.uuid4()), "Ambrosia artemisiifolia")
+ambrosiaTrifida = (str(uuid.uuid4()), "Ambrosia trifida")
+Ambrosiapsilostachya = (str(uuid.uuid4()), "Ambrosia psilostachya")
+
+cur.execute(f"""SET search_path TO "{NACHET_SCHEMA}";""")
+conn.commit()
+
+# Query to insert a seed
+query = "INSERT INTO seeds (id,name) VALUES (%s,%s),(%s,%s),(%s,%s),(%s,%s),(%s,%s),(%s,%s),(%s,%s),(%s,%s),(%s,%s),(%s,%s),(%s,%s),(%s,%s),(%s,%s),(%s,%s),(%s,%s)"
```

Comment:
this query would be unnecessary if the uuid is generated automatically for teh uuid column. then, instead of creating a variable for each species you can stick to a single array/tuple of names and pass that as data.

Reply:
```suggestion
    seeds = (
        "Brassica napus",
        "Brassica juncea",
        "Cirsium arvense",
        "Cirsium vulgare",
        "Carduus nutans",
        "Bromus secalinus",
        "Bromus hordeaceus",
        "Bromus japonicus",
        "Lolium temulentum",
        "Solanum carolinense",
        "Solanum nigrum",
        "Solanum rostratum",
        "Ambrosia artemisiifolia",
        "Ambrosia trifida",
        "Ambrosia psilostachya",
    )
```